### PR TITLE
add iotty outlet docs

### PR DESCRIPTION
**Pull request by shapournemati-iotty**
## Proposed change

Add iotty outlet details.

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logos and icons in [Brands repository](https://github.com/home-assistant/brands).
- [x] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/132912
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

## Summary by CodeRabbit

- **New Features**
	- Added support for the "Cover" category and platform in the documentation.
	- Introduced a new code owner for the integration.

- **Documentation**
	- Enhanced clarity and structure of the iotty integration documentation.
	- Reformatted sections for supported devices, improving readability and organization.
	- Expanded descriptions for the main switch entity to clarify its functionality.

### CODE REVIEW

File: source/_integrations/iotty.markdown [hunk 1]
```diff
@@ -3,46 +3,67 @@ title: iotty
...
+### iotty Shutter
+
+- [iotty Smart Shades Switch](https://iotty.uk/collections/frontpage/products/e2s-plus-smart-shades-switch-for-shutters-and-blinds)
+- [iotty Plus Interruttore Intelligente per Tende e Tapparelle](https://iotty.it/collections/prodotti-singoli/products/i3s-plus-interruttore-intelligente-per-tende-e-tapparelle) (currently only available for the Italian market)
+
+### iotty Outlet
+
+- [iotty Smart Outlet - Italy](https://iotty.it/collections/prodotti-singoli/products/oit-plus-presa-intelligente)
+- [iotty Smart Outlet - Germany](https://iotty.fr/collections/prodotti-singoli/products/ofr-plus-prise-intelligente)
+- [iotty Smart Outlet - France](https://iotty.de/collections/prodotti-singoli/products/ode-plus-smarte-steckdose)
```
[the remark this answers is not in the document]
Comment (shapournemati-iotty) [reply source/_integrations/iotty.markdown]: Fixed broken links

[the remark this answers is not in the document]
Comment (shapournemati-iotty) [reply source/_integrations/iotty.markdown]: Can you double-check? I can't see these links in the latest version (check last commit please)

File: source/_integrations/iotty.markdown [hunk 2]
```diff
@@ -41,22 +41,28 @@ US version:
...
 ### iotty Shutter
 
 - [iotty Smart Shades Switch](https://iotty.uk/collections/frontpage/products/e2s-plus-smart-shades-switch-for-shutters-and-blinds)
 - [iotty Plus Interruttore Intelligente per Tende e Tapparelle](https://iotty.it/collections/prodotti-singoli/products/i3s-plus-interruttore-intelligente-per-tende-e-tapparelle) (currently only available for the Italian market)
 
+### iotty Outlet
+
+- [iotty Smart Outlet - Italy](https://iotty.it/collections/prodotti-singoli/products/oit-plus-presa-intelligente)
+- [iotty Smart Outlet - Germany](https://iotty.fr/collections/prodotti-singoli/products/ofr-plus-prise-intelligente)
+- [iotty Smart Outlet - France](https://iotty.de/collections/prodotti-singoli/products/ode-plus-smarte-steckdose)
```
Comment (silamon) [inline source/_integrations/iotty.markdown]: You probably swapped these links.

**Review by silamon — CHANGES_REQUESTED** [review]

File: source/_integrations/iotty.markdown (hunk 2 above)
Comment (shapournemati-iotty) [reply source/_integrations/iotty.markdown]: Nice catch, thanks a lot! I have fixed this

**Review by silamon — APPROVED** [review]

**Pull request merged**